subiquity/tests/api/test_api.py

```diff
@@ -1561,6 +1584,27 @@ async def test_upcs_flow(self):
                 await inst.get('/ubuntu_pro/contract_selection/wait')
 
 
+class TestAutoinstallServer(TestAPI):
+    @timeout(2)
+    async def test_make_view_requests(self):
+        cfg = 'examples/simple.json'
+        extra = [
+            '--autoinstall', 'examples/autoinstall-short.yaml',
+            '--source-catalog', 'examples/install-sources.yaml',
+        ]
+        async with start_server(cfg, extra_args=extra) as inst:
+            view_request_unspecified = await inst.get('/locale')
+            self.assertEqual('en_US.UTF-8', view_request_unspecified)
+
+            view_request_no = await inst.get(
+                    '/locale', headers={'x-make-view-request': 'no'})
+            self.assertEqual('en_US.UTF-8', view_request_no)
+
+            view_request_yes = await inst.get(
+                    '/locale', headers={'x-make-view-request': 'yes'})
+            self.assertIsNone(view_request_yes)
```

Comment:
do you have any way to test the value of the x-skip header in the response?

Reply:
if the answer is "no" that's fine, having any tests >> having no tests!

Reply:
I debated doing so.  I'll spend a little time on it and explore.

Reply:
Implemented.